app/presenters/hyrax/displays_image.rb

```diff
@@ -38,8 +41,17 @@ def iiif_endpoint(file_id)
         )
       end
 
+      def lookup_original_file_id
+        result = original_file_id
+        if result.blank?
+          Rails.logger.warn "original_file_id for #{id} not found, falling back to Fedora."
+          result = ActiveFedora::File.uri_to_id(::FileSet.find(id).current_content_version_uri)
```

Comment:
Does this need a fallback in the case that `original_file.versions.present?` would fail?  `FileSetPresenter#original_file_id` in this PR is doing that at least.
I also wonder if both of these cases are getting tested.

Reply:
I DRYed this up by moving it to the VersioningService.